src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -921,6 +921,9 @@ namespace cryptonote
           }
         }
 
+        if (arg.blocks.empty())
+          goto skip;
```

Comment:
I once worked on a project where `goto`s were banned. One of them C programming types wrote a giant set of `do { if (...) break; } while (0);` conditions instead. Just let the programmers have their `goto`s!